data/lm/README.rst

```diff
@@ -7,22 +7,24 @@ You can download the LibriSpeech corpus with the following commands:
 
     wget http://www.openslr.org/resources/11/librispeech-lm-norm.txt.gz
 
+|
 
-Then use the `generate_lm.py` script to generate `lm.binary` and `vocab-500000.txt`.
+| Then use the ``generate_lm.py`` script to generate ``lm.binary`` and ``vocab-500000.txt``.
+| As input you can use a plain text (e.g. ``file.txt``) or gzipped (e.g. ``file.txt.gz``) text file with one sentence in each line.
```

Comment:
Why are you using a line block here?

Reply:
To create a empty line between the different commands.

Reply:
The way you write paragraphs in reStructuredText is by skipping a blank line, not by using line blocks: https://docutils.sourceforge.io/docs/user/rst/quickstart.html

Reply:
I wanted two blank lines instead of one, for better separation. Multiple blank lines got compressed to only a single blank line, thats why i tried it with a empty line block.

Reply:
Please just use paragraphs. Two goals of using a simple markup language like Markdown or reST are: to make both the source and the formatted output easy to read, and to avoid having to worry about the small details of presentation.

Reply:
Looks the same to me, are you sure you pushed the changes? Still see a bunch of line blocks.

Reply:
I did remove the empty lineblocks. 
I left the other lineblocks to structure the text.

Or did you want me to remove all of the lineblocks?

Reply:
I want you to just use paragraphs (the way the file is currently structured, before this PR), instead of using line blocks to fine tune paragraph whitespace.

Reply:
ok, i thought this was only about the empty lineblocks for double line spacings.